langchain/callbacks/tracers/evaluation.py

```diff
@@ -0,0 +1,50 @@
+"""A tracer that runs evaluators over completed runs."""
+from concurrent.futures import Future, ThreadPoolExecutor, wait
+from typing import Any, List, Optional, Sequence, Set, Union
+from uuid import UUID
+
+from langchainplus_sdk import LangChainPlusClient, RunEvaluator
+
+from langchain.callbacks.tracers.base import BaseTracer
+from langchain.callbacks.tracers.schemas import Run
+
+
+class EvaluatorCallbackHandler(BaseTracer):
```

Comment:
Thoughts on this? It lets us start running feedback immediately rather than waiting for the entire session to complete, which is nice in one respect. Although it may lead to some undesired effects if people get rate limited and have a higher concurrency level perhaps?